### DISCUSSION TITLE
Emit scrolling-related events

### DISCUSSION THREAD

**Discussion by xzilja** [idiotWu/smooth-scrollbar#364] [Ideas]
I'm writing some logic that needs to be started on scroll start and ended when scrolling ends / becomes idle.

I can detect scroll start with `scrolbar.addListener` listener, but I can't seem to find anything that indicates "stop" event i.e. when scrollbars become idle and user is no longer pressing on the screen / using mouse wheel.

Is event like this exposed? If not what would be right approach here?

**Comment by xzilja** [comment]
https://github.com/idiotWu/smooth-scrollbar/issues/126 this works, perhaps it could be a good idea to expose end event however?

**Comment by dolphin-wood** [comment]
Maybe we can write a plugin that emits all the related events (e.g. `scroll-start, scroll-end, etc.`). I'll consider adding this in the next major release, thanks!
